gradle.properties

```diff
@@ -0,0 +1,2 @@
+ossrhUsername=dummyId
```

Comment:
Do we need this file to be in source control? Or should we make it gradle.properties.sample?  This is to avoid the real creds being checked in by mistake.